cime_config/tests.py

```diff
@@ -650,6 +650,9 @@
             "SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.scream-output-preset-3",
             "ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-output-preset-4",
             "ERS_D_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-rad_frequency_2--scream-output-preset-5",
+            "ERS_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-small_kernels--scream-output-preset-5",
+            "ERS_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-small_kernels_p3--scream-output-preset-5",
```

Comment:
Maybe not for now, but I wonder if we could add these to other existing tests, and keep the v1 lowres suite compact. I like that it's currently a handful tests (runs quickly). Yes, they're all ne4 tests, so they run fast, but I'm also thinking about the build time...

@jgfouca Can we make all these tests use the same sharedlib phase? I know we can make tests use the same exec, but that may not be possible here,  since the small kernels are build time differences (although the first 4 tests could maybe be made to use the same exec?).